pwa/src/templates/templateParts/footer/FooterTemplate.tsx

```diff
@@ -78,106 +106,96 @@ export const FooterTemplate: React.FC<FooterTemplateProps> = ({ layoutClassName
+                </Icon>
+                Github
               </Link>
             </li>
           </ul>
 
           <ul className={styles.list}>
             <div className={styles.heading}>Links</div>
 
-            <li
-              onClick={() => {
-                open("https://commonground.nl/");
-              }}
-            >
-              <Link icon={<CommongroundLogo />} iconAlign="start">
+            <li>
+              <Link href="https://commonground.nl/">
+                <Icon className="utrecht-icon--conduction-start">
+                  <CommongroundLogo />
+                </Icon>
                 Common ground
               </Link>
             </li>
 
-            <li
-              onClick={() => {
-                open("https://haven.commonground.nl/");
-              }}
-            >
-              <Link icon={<HavenLogo />} iconAlign="start">
+            <li>
+              <Link href="https://haven.commonground.nl/">
+                <Icon className="utrecht-icon--conduction-start">
+                  <HavenLogo />
+                </Icon>
                 Haven
               </Link>
             </li>
 
-            <li
-              onClick={() => {
-                open("https://designsystem.gebruikercentraal.nl/");
-              }}
-            >
-              <Link icon={<ExternalLinkIcon />} iconAlign="start">
+            <li>
+              <Link href="https://designsystem.gebruikercentraal.nl">
+                <Icon className="utrecht-icon--conduction-start">
+                  <ExternalLinkIcon />
+                </Icon>
                 NL design
               </Link>
             </li>
 
-            <li
-              onClick={() => {
-                open("https://forumstandaardisatie.nl/");
-              }}
-            >
-              <Link icon={<ForumStandaardisatieLogo />} iconAlign="start">
+            <li>
+              <Link href="https://forumstandaardisatie.nl/">
+                <Icon className="utrecht-icon--conduction-start">
+                  <ForumStandaardisatieLogo />
+                </Icon>
                 Forum standaardisatie
               </Link>
             </li>
 
-            <li
-              onClick={() => {
-                navigate("#");
-              }}
-            >
-              <Link icon={<ExternalLinkIcon />} iconAlign="start">
+            <li>
+              <Link href="#">
+                <Icon className="utrecht-icon--conduction-start">
+                  <ExternalLinkIcon />
+                </Icon>
                 {t("Privacy declaration")}
               </Link>
             </li>
 
-            <li
-              onClick={() => {
-                open(window.sessionStorage.getItem("ADMIN_DASHBOARD_URL") ?? "#");
-              }}
-            >
-              <Link icon={<FontAwesomeIcon icon={faCircleUser} />} iconAlign="start">
+            <li>
+              <Link href={window.sessionStorage.getItem("ADMIN_DASHBOARD_URL") ?? "#"}>
```

Comment:
Je zou er een placeholder link van kunnen maken als de URL niet beschikbaar is:
`placeholder={!window.sessionStorage.getItem("ADMIN_DASHBOARD_URL")}`